src/Icon/old-glyphs.json

```diff
@@ -111,7 +111,6 @@
   "sticker": 59815,
   "starNo": 59813,
   "stack": 59812,
-  "search": 59792,
```

Comment:
Leave it until we drop the old icons altogether

Reply:
If I leave this here, the mdi icon will be re-written by it

Reply:
Make it so the new icons takes precedence over the old ones